CodeStarterCamp_Week2/LottoDatabase.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  LottoDatabase.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by QL Jeong on 2023/08/25.
+//
+
+import Foundation
+
+@discardableResult func addWinNumbers(_ winNumbers: [Int]) -> Int {
```

Comment:
`discardableResult` attribute를 사용하셨네요!
return 값이 있는 이유는 로또 번호를 추가하는 도중에 round를 확인하기 위함일까요? 💭

indent를 맞추기 위해 attribute를 사용할 때 줄바꿈을 해주는 컨벤션도 있습니다 👀
```suggestion
@discardableResult
func addWinNumbers(_ winNumbers: [Int]) -> Int {
```

Reply:
네 attribute 중에는 전달값이 필요한 것들도 있어서 라인을 나누어서 쓰기도 합니다만,
저는 2개 이상을 할당 해야 하거나 전달값 때문에 시인성을 올리기 위한 내용이 아니면
해당하는 method 가 어떤 속성을 할당받고 있는지를 좀더 눈에 띄도록 일부러 indenting 을 일그러 트려 놓은 것 입니다.

fileprivate 등도 같은 형태로 줄을 나누어서 할당이 되지만 명시적으로 보기 위해서 앞에 놓는 것과 같은 이유입니다.